superset-frontend/src/features/alerts/AlertReportModal.tsx

```diff
@@ -1581,7 +1581,7 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
           key="schedule"
         >
           <AlertReportCronScheduler
-            value={currentAlert?.crontab || ALERT_REPORTS_DEFAULT_CRON_VALUE}
+            value={currentAlert?.crontab || ''}
```

Comment:
I don't have much context here but this does look problematic. Is the default being completely ignored? I don't see other uses of the variable. Maybe @eschutho can help here.

Reply:
The default is being set when a new alert/report is generated, so the user should always open the modal to a valid schedule input to start.

Reply:
Given that we are validating empty CRON values, this change is fine.